app/controllers/concerns/course/forum/topics_hide_concern.rb

```diff
@@ -0,0 +1,25 @@
+module Course::Forum::TopicsHideConcern
```

Comment:
Yes, but when I use TopicControllerHideConcern, there will be an uninitialized constant error in the browser sometimes.